ci/pipelines/deploy-to-staging.yml

```diff
@@ -460,7 +477,82 @@ jobs:
         trigger: true
       - set_pipeline: deploy-to-staging
         file: deploy-to-staging-pipeline-definition/ci/pipelines/deploy-to-staging.yml
-        
+
+  - name: deploy-carbon-relay
```

Comment:
Can we name this `deploy-carbon-relay-to-staging` to be consistent with the other apps?

Reply:
This has been applied: https://cd.gds-reliability.engineering/teams/pay-deploy/pipelines/deploy-to-staging/jobs/deploy-carbon-relay-to-staging/builds/1